bin/main.js

```diff
@@ -271,9 +270,10 @@ export async function main(nodeProcess) {
     nodeProcess.exit(0);
   } catch (err) {
     if (err.code === 'ENOENT') {
-      nodeProcess.stderr.write('marked: output to ' + err.path + ': No such directory');
+      nodeProcess.stderr.write('marked: ' + err.path + ': No such directory');
```

Comment:
```suggestion
      nodeProcess.stderr.write('marked: ' + err.path + ': No such file or directory');
```

Should this mention file or directory since its not always a directory?